docs/document-scanner/options.md

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 99
+title: Options
+---
+
+# Document Scanner Options
+
+A `DocumentScannerOptions` object is used to configure the behavior of the document scanner. This interface provides several optional properties that allow you to enable or disable certain features, set the mode of the scanner and choose the format for the results.
+
+| Property               | Type                  | Optional | Description                                                             |
+| :--------------------- | :-------------------- | -------- | :---------------------------------------------------------------------- | --- |
+| `pageLimit`            | `number`              | Yes      | The maximum number of pages that can be scanned.                        |
+| `galleryImportAllowed` | `boolean`             | Yes      | Allow the user to choose images from gallery or force a new photograph. |
+| `scannerMode`          | `ScannerModeOptions`  | Yes      | Sets the mode of the scanner.                                           |     |
+| `resultFormats`        | `ResultFormatOptions` | Yes      | Sets the format of the scanned document results.                        |     |
```

Comment:
![image](https://github.com/infinitered/react-native-mlkit/assets/22041394/a12f9a82-e32d-4f3a-8651-19f131b90607)

I love a well-formatted markdown table

Reply:
Yeah IDK why my auto format does that, removing spaces I guess